alt text updated (#19407)
mdn/content: files/en-us/games/tutorials/2d_breakout_game_pure_javascript/move_the_ball/index.md

Save your code again and try it in your browser. This works OK, although it appears that the ball is leaving a trail behind it:

> ![a white background with a thick blue line drawn at 45 degrees from the right and not touching the base](ball-trail.png)

Reviewer: ```suggestion
![A thick blue line going up and to the right at 45 degrees](ball-trail.png)
```
